draft-ietf-quic-transport.md

```diff
@@ -2070,91 +2069,107 @@ following layout:
 This design ensures that a stateless reset packet is - to the extent possible -
 indistinguishable from a regular packet with a short header.
 
-A server generates a random 18-octet Destination Connection ID field.  For a
-client that depends on the server including a connection ID, this will mean that
-this value differs from previous packets.  Ths results in two problems:
+The message consists of a header octet, followed by random octets of arbitrary
+length, followed by a Stateless Reset Token.
+
+The endpoint SHOULD send a packet with a short header.
```

Comment:
This seems like a strange SHOULD. The Stateless Reset packet is defined here with a short header. How can this be a long header?